graphql_compiler/scripts/generate_test_sql/__init__.py

```diff
@@ -0,0 +1,18 @@
+import itertools
+import random
+
+from .animals import get_animal_generators
+from .species import get_species_generators
+
+
+random.seed(0)
+
+
+if __name__ == '__main__':
+    sql_generators = [
+        get_species_generators,
+        get_animal_generators,
+    ]
+    sql_generator_strings = []
+    for generator in sql_generators:
+        print('\n'.join(generator()))
```

Comment:
You'll need to do `sys.stdout.write()` here, the linter doesn't like the `print`. Also, I thought we were going to go with printing things as they are generated instead of printing one massive string at once?